CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Move codebase to github
+- Move doc to readthedocs.io
```

Comment:
Maybe reference the MR number? I think it's something github also has: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls
```suggestion
- Move doc to readthedocs.io #6
```

Reply:
Yes, nice catch ! Done